File: src/com/github/legendofmcpe/pocketbot/packet/raknet/OpenConnectionRequest1.java
package com.github.legendofmcpe.pocketbot.packet.raknet;


public class OpenConnectionRequest1 extends SentRaknetPacket{
	private int mtu = -1;
	public OpenConnectionRequest1(int mtu){
		super(mtu + 18);
	}
	@Override
	public byte getPid(){
		return OPEN_CONNECTION_REQUEST_1;
	}
	@Override
	public void encode(){
		bb().put(MAGIC_BYTES);
		bb().put(RAKNET_VERSION);
		for(int i = 0; i < mtu; i++){
			bb().put((byte) 0x00);
		}
	}
}

File: src/LegendOfMCPE/PocketBot/network/raknet/OpenConnectionRequest1.php
<?php

namespace LegendOfMCPE\PocketBot\network\raknet;

class OpenConnectionRequest1 extends RaknetPacket{
	private $mtu;
	public function __construct($order){
		$mtus = [1447, 1447, 1447, 1447, 1155, 1155, 1155, 1155, 531, 531, 531, 531, 531];
		if(!isset($mtus)){
			throw new \RuntimeException;
		}
		$this->mtu = $mtus[$order];
	}
	public function getPid(){
		return self::OPEN_CONNECTION_REQUEST_1;
	}
	public function encode(){
		$this->writeChar(self::OPEN_CONNECTION_REQUEST_1);
		$this->writeLong(self::MAGIC);
		$this->writeChar(self::RAKNET_PROTOCOL_VERSION);
		$this->write(str_repeat("\x00", $this->mtu));
	}
}
